### PR TITLE
311 Section

### PR DESCRIPTION
**Pull request by ifsimicoded**

### CODE REVIEW

File: src/css/modules/_ListLink.scss [hunk 1]
```diff
@@ -1,34 +1,13 @@
 .coa-ListLink {
-  @include box-verticalcenter();
-  background-color: $color-white;
+  @include box-verticalcenter(7.5rem);
```
Comment (mateoclarke) [inline src/css/modules/_ListLink.scss]: 🔥

File: src/js/modules/ContactEmail.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 //TODO: import correct svg
-import AirplaneSVG from 'js/svg/Airplane';
+import EnvelopSVG from 'js/svg/EnvelopO';
```
Comment (mateoclarke) [inline src/js/modules/ContactEmail.js]: `EnvelopO`? typo?

File: src/js/modules/ContactHours.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React, {Component} from 'react';
 import { FormattedMessage, FormattedTime } from 'react-intl';
 import {sortBy, findIndex} from 'lodash';
 //TODO: import correct svg
-import AirplaneSVG from 'js/svg/Airplane';
+import ClockSVG from 'js/svg/ClockO';
```
Comment (mateoclarke) [inline src/js/modules/ContactHours.js]: `ClockO`, typo?

File: src/js/modules/ContactEmail.js (hunk 2 above)
Comment (ifsimicoded) [reply src/js/modules/ContactEmail.js]: it was envelop-O -- vs filled in... so here we are

**Pull request merged**